modules.json

```diff
@@ -0,0 +1,486 @@
+{
+  "tenpin":
+  [
+    {
+      "name": "accelerometer",
+      "package": "accel-mma84",
+      "chip": "MMA8452Q",
+      "datasheet": "http://www.freescale.com/files/sensors/doc/data_sheet/MMA8452Q.pdf",
+      "image": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg",
```

Comment:
Interesting, but I don't think it's going to come up; we don't have them for the existing tenpin modules, and we're not planning to add new ones soon.